bentoml/deployment/aws_lambda/lambda_app.py

```diff
@@ -22,13 +22,15 @@
 
 # Set BENTOML_HOME to /tmp directory due to AWS lambda disk access restrictions
 os.environ['BENTOML_HOME'] = '/tmp/bentoml/'
-from bentoml.deployment.aws_lambda.utils import download_artifacts_for_lambda_function
+
+from bentoml.utils.s3 import download_directory_from_s3
 from bentoml.bundler import load_bento_service_class
 
-bento_name = os.environ.get('BENTO_SERVICE_NAME')
-s3_bucket = os.environ.get('S3_BUCKET')
-artifacts_prefix = os.environ.get('ARTIFACTS_PREFIX')
-tmp_artifacts_dir = '/tmp/bentoml'
+bento_name = os.environ.get('BENTOML_BENTO_SERVICE_NAME')
```

Comment:
use `os.environ['BENTOML_BENTO_SERVICE_NAME']` instead, to ensure a KeyError is rasied, when env var is not set